tests/scripts/depends.py

```diff
@@ -500,15 +500,27 @@ def __init__(self, options, conf):
         # and padding modes are exercised separately) information by parsing
         # cipher.h, as the information is not readily available in mbedtls_config.h.
         cipher_info = CipherInfo()
-        # Find block cipher chaining and padding mode enabling macros by name.
-        cipher_chaining_symbols = self.config_symbols_matching(r'MBEDTLS_CIPHER_MODE_\w+\Z')
+
+        # Get block cipher chaining modes. Do not select ECB, it is always enabled.
+        cipher_modes_filter = re.compile(r'PSA_WANT_ALG_(?!ECB|STREAM|CCM)\w+\Z')
+        cipher_chaining_symbols = {symbol
+                                   for alg, symbol in algs.items()
+                                   if alg.can_do(crypto_knowledge.AlgorithmCategory.CIPHER)
+                                   if re.match(cipher_modes_filter, symbol)}
+
+        # Find block padding mode enabling macros by name.
         cipher_padding_symbols = self.config_symbols_matching(r'MBEDTLS_CIPHER_PADDING_\w+\Z')
+
         self.domains = {
             # Cipher IDs, chaining modes and padding modes. Run the test suites.
             'cipher_id': ExclusiveDomain(cipher_info.base_symbols,
                                          build_and_test),
+
+            # XTS is not supported via the PSA API.
```

Comment:
Not yet supported. Please add a link to https://github.com/Mbed-TLS/mbedtls/issues/6384, that will help us find non-obvious places that need updating when we add XTS support.